src/data/fintype/basic.lean

```diff
@@ -40,8 +40,14 @@ fintype.complete x
 @[simp] lemma coe_univ : ↑(univ : finset α) = (set.univ : set α) :=
 by ext; simp
 
-lemma univ_nonempty [h : nonempty α] : (univ : finset α).nonempty :=
-nonempty.elim h (λ x, ⟨x, mem_univ x⟩)
+lemma univ_nonempty_iff : (univ : finset α).nonempty ↔ nonempty α :=
```

Comment:
Should this be `nonempty_univ` since without projections it's `nonempty univ`?

Reply:
It is `univ_nonempty` for sets, so I'd prefer to leave it as is. We can rename both to `nonempty_univ`, but this should be done in a separate PR.